ops/ensure_kibana_configuration/dot.kibana.analyzer

```diff
@@ -0,0 +1 @@
+"{\".kibana\":{\"settings\":{\"index\":{\"number_of_shards\":\"1\",\"number_of_replicas\":\"1\",\"provided_name\":\".kibana\"}}}}"
```

Comment:
not needed

Reply:
was generated while exporting the database, when curl import/export get implemented this should disappear.